DEV.md

```diff
@@ -13,8 +13,9 @@
 
 - Requires that you are listed as a maintainer for this npm module
 
-* Commit and push changes
-* Login: `npm login`
 * Make sure you can run build successfully: `npm run build`
 * Update version: `npm version patch`
-* Publish: `npm publish`
+* Push to main: `git push origin main`
+* Push tags: `git push origin --tags vX.X.X`
```

Comment:
two questions: 
1. Is the main branch protected? Should we create PRs for pushing tags, or is it safe to push directly to main? 
2. Should we specify the commands for pushing tags:
    - push all tags: `git push origin --tags`
    - push a single tag: `git push origin vX.X.X`

Reply:
1. Main is protected.
2. Good point! Changed this to push a single tag.